### PR TITLE
Open user profile from membership requests

### LINKED ISSUE #12945: Add ability to open a user profile from the 'Membership requests' screen in community
# Feature Issue

[comment]: # (Please replace ... with your information. Remove < and >)

### User Story

As a community admin, I want to be able to open profiles of users who requested membership in the community so that I can get more info about this user. 

### Description

Currently, an admin cannot open profiles from the `Membership requests` screen, so they cannot get more info about the user or contact them.

https://user-images.githubusercontent.com/67952253/146929962-24a80a06-4d16-43fe-943b-ce4f07d2f512.mp4

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-react/issues/12945

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/communities/requests_to_join.cljs [hunk 1]
```diff
@@ -11,19 +11,20 @@
...
+            [quo.components.animated.pressable :as animation]))
 
 (defn hide-sheet-and-dispatch [event]
   (>evt [:bottom-sheet/hide])
   (>evt event))
 
 (defn request-actions [community-id request-id]
   [react/view {:flex-direction :row}
-   [react/touchable-highlight {:on-press #(re-frame/dispatch [:communities.ui/accept-request-to-join-pressed community-id request-id])}
+   [animation/pressable {:on-press #(re-frame/dispatch [:communities.ui/accept-request-to-join-pressed community-id request-id])}
```
Comment (Parveshdhull) [inline src/status_im/ui/screens/communities/requests_to_join.cljs:23]: When `react/touchable-highlight` is used for button _(child)_, then its on-press is also calling `on-press` of `quo/list-item` _(parent)_
Seems, imports from `react` or `react-native` library doesn't work well with `react-native-gesture-handler` library

https://stackoverflow.com/a/59570332

File: src/status_im/ui/components/emoji_thumbnail/preview.cljs [hunk 2]
```diff
@@ -6,7 +6,8 @@
 (defn emoji-thumbnail [emoji color size]
   (when-not (string/blank? emoji)
     [react/view (styles/emoji-thumbnail-icon color size)
-     [react/text {:style (styles/emoji-thumbnail-icon-text size)} emoji]]))
+     [react/text {:style               (styles/emoji-thumbnail-icon-text size)
```
Comment (Parveshdhull) [inline src/status_im/ui/components/emoji_thumbnail/preview.cljs:9]: this part is not related to this issue. it will remove invalid prop type error message from the metro.

**Review by rasom — APPROVED** [review]

**Review by bitgamma — APPROVED** [review]

**Comment by churik** [comment]
e2e failures are not related to PR.
Tested basic flows, LGTM!

**Devices:**
- Xiaomi Mi Note 9 Pro (Android 10)
